code/modules/medical/computer/cloning.dm

```diff
@@ -447,12 +447,21 @@
 			break
 	return selected_record
 
+/obj/machinery/computer/cloning/process()
+	update_icon()
+
 /obj/machinery/computer/cloning/update_icon()
```

Comment:
Doing status overlays this way can be incredibly expensive, its for this reason the APC was once considered one of the laggiest objects in the game simply from update icon. Calling it every tick merely compounds this problem. 

See the measures in apc.dm's update_icon they had to take to avoid this being an issue, holding the icons in a list to be taken out and applied on the object as needed.